real-main/app_tests/models/user/test_dynamo.py

```diff
@@ -275,10 +284,23 @@ def test_set_user_details(user_dynamo):
             'verificationHidden': True,
             'birthday': birthday,
             'gender': gender,
+            'currentLocation': current_location,
+            'matchAgeRange': match_age_range,
+            'matchGenders': match_genders,
+            'matchLocationRadius': match_location_radius,
         },
     }
     assert resp == expected
 
+    # assert if accuracy is not set
+    current_location = {"latitude": Decimal('50'), "longitude": Decimal('50')}
+    resp = user_dynamo.set_user_details(
+        user_id,
+        current_location=current_location,
+    )
+    assert resp['currentLocation'] == current_location
+    assert resp['currentLocation'].get('accuracy') is None
```

Comment:
```suggestion
    assert 'accuracy' not in resp['currentLocation']
```

This suggestion is a slightly stricter assertion - it should ensure we aren't storing a null in dynamo